guides/getting_started.md

```diff
@@ -0,0 +1,113 @@
+
+
+### Installing Ruby
+
+Installing Ruby varies widely from architecture to architecture. There is a good overview on how to
+install Ruby on the [Ruby homepage](https://www.ruby-lang.org/en/documentation/installation/).
+
+Please make sure your ruby version is at least 2.2.2:
+
+```
+$ ruby -v
+ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
+```
+
+### Installing Rails
+
+Ruby comes with the package manager `rubygems`. If your Ruby is setup correctly, this should be as easy as running the following command:
+
+```
+$ gem install rails bundler
+```
+
+### Generating your host app
+
+It's important to understand that Solidus is a [Rails engine](http://guides.rubyonrails.org/engines.html), and that your host app simply requires that engine. The first step for installing Solidus is therefore generating a vanilla Rails application.
+
+Solidus will not work well with the `sqlite` database, and it runs best with the `postgresql` database.
+The ecosystem in general uses the `rspec` testing framework, so we skip the standard `minitest` installation. To generate your host app with these options, run the following command:
+
+```
+rails new my-shop -d postgresql --skip-test
+```
+
+Switch into your newly generated app directory:
+
+```
+cd my-shop
+```
+
+Now, run the tasks to create your development and test databases. If this does not work, please consult the postgres installation instructions for your architecture and the Rails Guide on [Configuring Rails applications](http://edgeguides.rubyonrails.org/configuring.html#rails-general-configuration).
+
+```
+bundle exec rake db:create db:migrate
+```
+
+You can find more detailed information about getting started with Rails in the [Rails Guides](http://guides.rubyonrails.org/getting_started.html)
+
+### Installing Solidus, Solidus Authentication (and RSpec)
+
+Installing Solidus into your app consists of adding the `solidus` and `solidus_auth_devise` gems to your `Gemfile`. Here is a working Gemfile for installing Solidus 2.0:
+
+```
+source 'https://rubygems.org'
+
+gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
+gem 'pg', '~> 0.18'
+gem 'puma', '~> 3.0'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
+gem 'jquery-rails'
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
+
+# Solidus and Solidus Authentication
+gem 'solidus', '~> 2.0'
+gem 'solidus_auth_devise'
+
+group :development, :test do
+  gem 'rspec-rails', '~> 3.5'
+end
+
+group :development do
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+```
+
+Now, run `bundle install` to download `solidus` and `solidus_auth_devise` into your new app, then run
+the following generators to get started:
+
+```
+bundle exec rails g spree:install
+bundle exec rails g solidus:auth:install
+bundle exec rake railties:install:migrations
```

Comment:
This is copied verbatim from the Readme. And you're right, that step can go, it doesn't do anything.